Doc/whatsnew/3.9.rst

```diff
@@ -186,6 +186,10 @@ Removed
   removed. They were deprecated since Python 3.7.
   (Contributed by Victor Stinner in :issue:`37320`.)
 
+* The ``threading.Thread.isAlive()`` method of ``threading.Thread`` has been
```

Comment:
```suggestion
* The ``~threading.Thread.isAlive()`` method of ``threading.Thread`` has been
```

should be rendered as "The isAlive() method of ...".

Reply:
<img width="859" alt="bpo-37894" src="https://user-images.githubusercontent.com/5110323/62873185-e8623500-bd59-11e9-8e62-6d7d7fe61b5e.png">

@vstinner

If we use the markdown you suggested, it will render in this way.
Is this rendering that you intend?

Reply:
How about just typing this way?
```markdown
The ``isAlive()`` method of .. 
```

Reply:
The screenshot was my mistake sorry.